content/rest-api/applications.md

```diff
@@ -5,6 +5,40 @@ weight: 2
 
 APIs for managing applications are currently available for developers to preview. During the preview period, the API may change without advance notice.
 
+## Retrieve an application

```

Comment:
currently, this endpoint is not public and we don't guarantee backward compatibility. Can you share more details about your use case to consider making the endpoint public?

Reply:
Using this endpoint one can get all the application details like **workflowIds**, **branches**, etc., that can be helpful for running a new build using the endpoint `/builds` which has these required parameters.

Reply:
I would love to see history of builds in the response, maybe together with artifacts links?